backend/CircleForms/Controllers/Authorization/OAuth/OAuthController.cs

```diff
@@ -116,6 +123,14 @@ public async Task<IActionResult> CompleteAuthentication()
 
         user = TransferMutableData(dbUser, user);
 
+        user.Token = new TokenResponse
+        {
+            AccessToken = accessToken,
+            ExpiresIn = 86400,
```

Comment:
Use osu's expiration instead

Reply:
osu expiration is not here

Reply:
![image](https://user-images.githubusercontent.com/8269193/160020260-759cde0a-9461-4ac4-97f5-c5fcf2100319.png)

Reply:
fq ths